source/dbo/Stored Procedures/Data Migration/uspMigrationDuplicateAlerts.sql

```diff
@@ -0,0 +1,18 @@
+﻿CREATE PROCEDURE [dbo].[uspMigrationDuplicateAlerts]
+	@MigrationRun INT = NULL
+AS
+	SELECT
+		mrr.MigrationNotificationId												AS 'MigrationNotificationId',
+		mrr.LegacyETSId															AS 'EtsId',
+		mrr.NTBSNotificationId													AS 'NtbsId',
+		mrr.NotificationDate													AS 'NotificationDate',
+		tbService.[Name]														AS 'TbService',
```

Comment:
The TB service name is already stored in Migration Run Results, so there's no need to calculate it again here.

Reply:
Ah, so it is! Thanks :-)